test/functional/p2p_getaddr_caching.py

```diff
@@ -21,21 +16,9 @@
     assert_equal,
 )
 
+# As defined in net_processing.
```

Comment:
0d04784af
```diff
-# As defined in net_processing.
-MAX_ADDR_TO_SEND = 1000
-MAX_PCT_ADDR_TO_SEND = 23
+MAX_ADDR_TO_SEND = 1000  # As defined in net.h
+MAX_PCT_ADDR_TO_SEND = 23  # As defined in net_processing.cpp
```

Reply:
Looks like a good suggestion,, but we can add more documentation later, didn't want to hold up the PR on this